### DISCUSSION TITLE
500s and timeout codes?

### DISCUSSION THREAD

**Discussion by ericadu** [deepgram/community#128] [General help]
Am seeing 500s (like below) and timeouts when making requests to the Deepgram Nova API this morning! 

We've been using Deepgram Nova in production for the past 3ish days without issues. https://status.deepgram.com/ seems all green. Unsure how to interpret these codes. 

Any advice?

```
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('https://api.deepgram.com/v1/listen?punctuate=true&tier=nova&model=general&language=en-US')
```
```
aiohttp.client_exceptions.ClientResponseError: 408, message='Request Timeout', url=URL('https://api.deepgram.com/v1/listen?punctuate=true&tier=nova&model=general&language=en-US') 
```

**Comment by lukeocodes** [comment]
Hey @ericadu, I've just made some requests to Nova and not had any errors. Can you confirm if this is still happening?
